instance/tasks.py

```diff
@@ -66,6 +67,16 @@ def spawn_appserver(instance_ref_id, mark_active_on_success=False, num_attempts=
 
 
 @db_task()
+def set_appserver_active(appserver_id):
+    """
+    Mark an AppServer as active.
+    """
+    logger.info('Retrieving AppServer: ID=%s', appserver_id)
```

Comment:
This log should be attached to the instance object? Like in https://github.com/open-craft/opencraft/blob/97a3b3a726c35b927706c2286b3b795e2869a262/instance/tasks.py#L58